daemon/attach.go

```diff
@@ -34,6 +34,10 @@ func (daemon *Daemon) ContainerAttach(prefixOrName string, c *backend.ContainerA
 		err := fmt.Errorf("Container %s is paused. Unpause the container before attach", prefixOrName)
 		return errors.NewRequestConflictError(err)
 	}
+	if container.IsRestarting() {
+		err := fmt.Errorf("Container %s is restarting. Wait until the container is running", prefixOrName)
```

Comment:
The error message looks a little strange. It's two sentences, and the first one has a period (`.`) after it, but the second one does not.

I notice that the error message is formatted the same way, but we should probably fix both.